src/__tests__/test.test.ts

```diff
@@ -0,0 +1,65 @@
+import * as iris from '../iris';
+
```

Comment:
Please rename this file to iris.test.ts

Test files have always the same name as the file we are testing + `.test`